requirements-micro.txt

```diff
@@ -0,0 +1,17 @@
+click
+apscheduler
+flask>=0.9            # for monitoring the web-service
+sqlalchemy>=1.1.9     # for database support
+configparser          # for parsing the config-file
+psutil                # for logging extra CPU-info
+colorhash             # for hashing a string into a color
+numpy                 # for computing median and other stats
+pytz                  # for timezone info
+tzlocal               # for figuring out the local timezone
```

Comment:
if you look at the requirements.txt you'll see that it has changed and is now frozen to 2.0.

Reply:
Right. I think that happend after I started working on this. Added.